packages/vite/bins/rw-vite-dev.mjs

```diff
@@ -11,9 +12,25 @@ const startDevServer = async () => {
     throw new Error('Could not locate your web/vite.config.{js,ts} file')
   }
 
+  // Tries to maintain the same options as vite's dev cli
+  // See here: https://github.com/vitejs/vite/blob/main/packages/vite/src/node/cli.ts#L103
+  // e.g. yarn rw dev web --fwd="--force"
```

Comment:
Thoughts on whether `--fwd` still makes sense? 

When we run `yarn rw dev` it runs both, so maybe the flag should be `webFwd`?